src/pages/General/experimentalFeatures/ExperimentalFeatures.js

```diff
@@ -0,0 +1,69 @@
+import i18n from '@dhis2/d2-i18n'
+import { Legend } from '@dhis2/ui'
+import isEmpty from 'lodash/isEmpty'
+import PropTypes from 'prop-types'
+import React, { useMemo } from 'react'
+import { CheckboxField } from '../../../components/field'
+import { MoreOptions } from '../../../components/moreOptions'
+import { getValidKeysList } from '../helper'
+import styles from './ExperimentalFeatures.module.css'
+import { featureList } from './feature-list'
+
+const CODE = 'experimentalFeatures'
+
+/**
+ * Create an object based on elements of "featureList"
+ * */
+const populateOptions = (list) =>
+    featureList?.reduce((options, e) => {
+        options[e.name] = !isEmpty(list) && list.includes(e.name)
+        return options
+    }, {})
+
+export const ExperimentalFeatures = ({ onChange, value, ...props }) => {
+    const experimentalValue = value[CODE]
+
+    const options = useMemo(
+        () => populateOptions(experimentalValue),
+        [experimentalValue]
+    )
+
+    const handleCheckbox = (e) => {
+        const optionsToSave = {
+            ...options,
+            [e.name]: e.checked,
+        }
+        onChange({ ...value, [CODE]: getValidKeysList(optionsToSave) })
+    }
```

Comment:
Wasn't my intention to conflict with Simonas suggestion here. I just forgot the `useCallback` in my example. You can decide if you want it in or not.